nucypher/blockchain/eth/actors.py

```diff
@@ -39,19 +39,19 @@
 from nucypher.blockchain.economics import TokenEconomics
 from nucypher.blockchain.eth.agents import (
     NucypherTokenAgent,
-    MinerAgent,
+    StakingEscrow,
```

Comment:
a little bit confused that other agents have `agent` in their names

Reply:
Thanks for noticing!
Fixed in #1046